### ISSUE TITLE
WCAG 1.4.10 Reflow issues

### ISSUE THREAD

**Issue by cmcmenamin** [sveltejs/sites#426]
For users with low vision, they may struggle to use svelte.dev/tutorial

At high browser zoom levels, which many devs with low vision use, the tutorial becomes very difficult to use due to non-dismissible banners, such as the header, and a banner for Ukraine.

Banners that can't be dismissed like this obscure a large percentage of the viewport, which means there is very little usable space in the viewport for the main content.

Here's an example that shows a large percentage of the main content being obscured by the header and banner:
<img width="1319" alt="A screenshot of svelte.dev showing the header and Ukraine banner covering large portions of the viewport, obscuring the main content" src="https://user-images.githubusercontent.com/112744591/213324858-34fbc2f2-5751-4c79-8bce-022f10b0168b.png">

I think making those banners/headers collapsable would make Svelte easier to learn for many visually impaired devs out there 😀

**Comment by cmcmenamin** [comment]
Note that this issue may not strictly constitute a WCAG 1.4.10 Reflow failure, but... it's kinda related to that WCAG Success Criterion.

**Comment by benmccann** [comment]
closing since the site has been rewritten. the current banner component has an ability to close it. please file at https://github.com/sveltejs/svelte.dev/issues/new if there are remaining issues
